### PR TITLE
Enable the "Speakers" block

### PR DESCRIPTION
**Pull request by emedvedev**
Adds the first confirmed speakers into the DB, enables the main page speakers block and the speakers page, hides the CFP button.

### CODE REVIEW

File: src/elements/dialogs/speaker-details.html [hunk 1]
```diff
@@ -179,6 +179,10 @@ <h3>{$ speakerDetails.sessions $}</h3>
       _getCloseBtnIcon(isLaptopViewport) {
         return isLaptopViewport ? 'close' : 'arrow-left';
       }
+
+      _hasJobTitle(speaker) {
```
Comment (helmuthb) [inline src/elements/dialogs/speaker-details.html]: Picky... but this is actually more like "`_hasNoJobTitle`"

File: data/dachfest-2018.json [hunk 2]
```diff
@@ -64,7 +64,141 @@
...
+    },
+    "1nfoverload" : {
+      "name" : "Daisy Tsang",
+      "country" : "Berlin, Germany",
+      "featured" : false,
+
+      "company" : "",
+      "title" : "",
+
+      "photoUrl" : "/images/people/speakers/1nfoverload.jpg",
```
Comment (helmuthb) [inline data/dachfest-2018.json]: This image is not here.

File: src/elements/dialogs/speaker-details.html (hunk 1 above)
Comment (emedvedev) [reply src/elements/dialogs/speaker-details.html]: Ohh, got lost in the intermediate edits. Thanks, good catch! Fixed :)

**Comment by emedvedev** [comment]
Yeah, Daisy doesn't have a pic yet, I've just put up a link just to make less changes afterwards. Thanks for the review!

**Pull request merged**